Sparky-core/src/sp/app/Input.cpp

```diff
@@ -2,6 +2,8 @@
 #include "Input.h"
 
 namespace sp {
+	std::map<uint, char> Input::normalChars = std::map<uint, char>();
+	std::map<uint, char> Input::shiftChars = std::map<uint, char>();
 
```

Comment:
Wont this be a problem with different style of keyboards? 
On a azerty keyboard the shiftkeys aren't at the same position as those on a qwerty.